Tests/LoggingTests/LocalLoggingTest.swift

```diff
@@ -114,15 +140,15 @@ private struct Struct1 {
 private struct Struct2 {
     func doSomething(context: Context) {
         var c = context
-        c.logLevel = .info // only effects from this point on
+        c.logLevel = .info  // only effects from this point on
         c.logger.info("Struct2::doSomething")
         self.doSomethingElse(context: c)
         c.logger.debug("Struct2::doSomething::end")
     }
 
     private func doSomethingElse(context: Context) {
         var c = context
-        c.logLevel = .debug // only effects from this point on
+        c.logLevel = .debug  // only effects from this point on
```

Comment:
I wonder if there's a way to prevent those, it's somewhat silly to insert a double space... maybe there's a bug for it already?

Reply:
Filed an issue about it https://github.com/swiftlang/swift-format/issues/868